### DISCUSSION TITLE
Planeación de Campaña de Vacunación. (Practica CSS)

### DISCUSSION THREAD

**Discussion by OmarVenturaP** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#268] [Ideas]
# Planeación de Campaña de Vacunación. (Practica CSS)

Buenas noches explores.
<img src="https://raw.githubusercontent.com/OmarVenturaP/Hola-Mundo/main/explorers.png" width="50px" alt="Hola Explorers">

Les comparto un poco de Finder relacionado a la Practica CSS. 
Se que muchos (como yo) tenemos ciertas dudas sobre la planeación de campaña de Vacunación. Así que me puse a investigar un poco y encontré algunos recursos que quizás pudieran ayudarles. Espero y alguno de ellos les pueda ayudar y no solamente en este proyecto. Saludos

<h2>Estrategias para hacer campañas de salud</h2>

https://www.saludiario.com/estrategias-para-hacer-campanas-de-salud/

<h2>¿Cómo planificar una campaña de publicidad?</h2>

https://www.ceupe.com/blog/como-planificar-una-campana-de-publicidad.html

<h2> Planeación Publicitaria Digital (En General)</h2>

https://blog.formaciongerencial.com/planificacion-publicitaria-digital/#:~:text=PROCESO%20DE%20PLANIFICACI%C3%93N%20PUBLICITARIA%20DIGITAL

<h2>Planificación y realización de una campaña Off y On line</h2>

https://www.marketing-xxi.com/planificacion-y-realizacion-de-una-campana-110.htm

**Comment by OscarPedroza** [comment]
Muchas gracias por tus aportaciones compañero.

**Comment by Paulina-Ceciliano** [comment]
Muchas gracias compañero! Me has ahorrado tiempecito con eso de la investigación
